omar-huanca/lesson-3/homework-1-guess-the-movie/GuessTheMovie.java

```diff
@@ -0,0 +1,96 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class GuessTheMovie {
+    private Scanner scanner;
+    private String movieTitle;
+    private char movieTitleAsterisks[];
+    private boolean isEntered[];
+    private int spaces;
+    private int movieLength;
+    private int score;
+
+    GuessTheMovie() {
+        this.scanner = new Scanner(System.in);
+        this.movieTitle = randomMovie();
+        this.movieTitleAsterisks = new char[movieTitle.length()];
+        this.isEntered = new boolean[movieTitle.length()];
+        this.spaces = 0;
+        this.score = 5;
+        movieEncrypted();
+        this.movieLength = movieTitle.length() - spaces;
+    }
+
+    private int randomMovieNumber() {
+        return (int) (Math.random() * 25);
+    }
+
+    private String randomMovie() {
```

Comment:
This works, but it is no t necessary to read the file each time you want to get a movie